code/modules/mob/living/simple_animal/abnormality/he/jangsan.dm

```diff
@@ -0,0 +1,288 @@
+//Code by Coxswain, EGO sprites by Sky_ and abnormality sprites by Mel
+/mob/living/simple_animal/hostile/abnormality/jangsan
+	name = "Jangsan Tiger"
+	desc = "A monster that eats children. Reforms its face for a friendly image"
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "jangsan_idle"
+	icon_living = "jangsan_idle"
+	var/icon_aggro = "jangsan"
+	faction = list("hostile")
+	speak_emote = list("growls")
+	pixel_x = -16
+	base_pixel_x = -16
+	ranged = TRUE
+	maxHealth = 1200
+	health = 1200
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.5, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 2)
+	see_in_dark = 10
+	stat_attack = HARD_CRIT
+	speed = 4
+	move_to_delay = 7
+	threat_level = HE_LEVEL
+	can_breach = TRUE
+	start_qliphoth = 3
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = 45,
+						ABNORMALITY_WORK_INSIGHT = 45,
+						ABNORMALITY_WORK_ATTACHMENT = 45,
+						ABNORMALITY_WORK_REPRESSION = 45
```

Comment:
This would make it way too good for all attributes.
Consider making it different?

Reply:
Numbers are quite malleable due to influence by stats, and I originally had variability between the work types. Want to see feedback from test merge before I change these as the work rates felt quite low in local testing